app/src/main/java/org/fossasia/openevent/general/event/EventApi.kt

```diff
@@ -11,4 +13,7 @@ interface EventApi {
     @GET
     fun getEvent(id: Long): Single<Event>
 
+    @GET("events/{identifier}/tickets")
+    fun getTickets(@Path("identifier") identifier: String): Flowable<List<Ticket>>
```

Comment:
This should be Single

Reply:
Actually this is where travis was showing error. I changed it to Flowable from Single and it passed.

Reply:
Okay :+1: